BoxBrowseSDK/BoxBrowseSDK/BOXFolderViewController.m

```diff
@@ -90,16 +94,34 @@ - (void)viewDidLoad
 
     [self setupForSearch];
     
-    UILabel *emptyLabel = [[UILabel alloc] init];
-    emptyLabel.textAlignment = NSTextAlignmentCenter;
-    emptyLabel.font = [UIFont boldSystemFontOfSize:15.0f];
-    emptyLabel.textColor = [UIColor colorWithRed:180.0f/255.0f green:179.0f/255.0f blue:180.0f/255.0f alpha:1.0f];
-    emptyLabel.backgroundColor = [UIColor clearColor];
-    emptyLabel.contentMode = UIViewContentModeCenter;
-    emptyLabel.numberOfLines = 0;
-    emptyLabel.autoresizingMask = UIViewAutoresizingFlexibleWidth;
-    emptyLabel.text = NSLocalizedString(@"Loading…", @"Loading message shown while content is being fetched.");
-    self.tableView.backgroundView = emptyLabel;
+    self.activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+    self.activityIndicator.hidesWhenStopped = YES;
+    [self.activityIndicator startAnimating];
+
+    self.folderInfoLabel = [[UILabel alloc] init];
+    self.folderInfoLabel.textAlignment = NSTextAlignmentCenter;
+    self.folderInfoLabel.font = [UIFont boldSystemFontOfSize:15.0f];
+    self.folderInfoLabel.textColor = [UIColor colorWithRed:180.0f/255.0f green:179.0f/255.0f blue:180.0f/255.0f alpha:1.0f];
+    self.folderInfoLabel.backgroundColor = [UIColor clearColor];
+    self.folderInfoLabel.contentMode = UIViewContentModeCenter;
+    self.folderInfoLabel.numberOfLines = 0;
+    self.folderInfoLabel.text = @"";
+    self.folderInfoLabel.hidden = YES;
+    self.folderInfoLabel.autoresizingMask = UIViewAutoresizingFlexibleWidth;
+}
+
+- (void)viewWillAppear:(BOOL)animated
```

Comment:
The label/activity indicator is misplaced when I set the frames in layout subviews instead:

viewWillAppear:
![screen shot 2015-09-17 at 11 23 29 am](https://cloud.githubusercontent.com/assets/3188130/9942203/adefd5d8-5d2e-11e5-92b1-df300efeec2f.png)

viewDidLayoutSubviews:
![screen shot 2015-09-17 at 11 22 37 am](https://cloud.githubusercontent.com/assets/3188130/9942198/aaf0158c-5d2e-11e5-95cd-28559d12ab5b.png)